features/tab-access-tabs.feature

```diff
@@ -0,0 +1,79 @@
+Feature: Get or set TabStops
+  In order to customize tab stops
+  As a python-docx developer
+  I need a TabStops object with read/write tab_stop properties
+
+
+  @wip
+  Scenario Outline: Get tab_stop objects
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops contain tab_stop objects
+      
+    Examples:  values for tab_stop elements 
+      | value |
+      | some  |
```

Comment:
This one is redundant. Let's drop it. I don't see how this is testing anything that's not covered by len(), iter, or indexed access.